### PR TITLE
Giving choice - npm or yarn.

### PR DESCRIPTION
**Pull request by NTillmann**
Fixing issue raised in #782.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,10 +7,10 @@ See the official [prepack.io](http://prepack.io) website for an introduction and
 
 ## How to use Prepack
 
-Install the CLI via yarn,
+Install the CLI via npm,
 
 ```bash
-$ yarn global add prepack
+$ npm install -g yarn
```
Comment (cblappert) [inline README.md]: This installs yarn not prepack, `npm install -g prepack` installs prepack

**Comment by NTillmann** [comment]
Updated README. Using npm for _using_ Prepack should be fine. You only need all the scripts for further development work. I added a note describing that.

**Comment by cblappert** [comment]
In this case we should also change the `package.json` to replace `npm` in all the scripts with `yarn` (e.g. `validate`, `prepublish`).

**Comment by NTillmann** [comment]
Okay, you one really needs yarn to develop.

**Pull request closed without merging**